ADRs/README.md

```diff
@@ -72,8 +72,10 @@ To contribute a new ADR, please follow these steps:
 3. Fill in the necessary information in the new ADR file, following the template structure.
 4. Use the large language model prompts, if desired, to generate or refine the ADR content.
 5. Submit a pull request to merge your ADR branch into the main branch, including a brief description of the problem and solution.
```

Comment:
I think we should clarify a bit around who is expected to do what. I'm noticing that these steps should probably addressed to different roles (author, reviewer/maintainer); something like:

5. Author: Open a PR as a draft ...
6. Author: Add a reference to the ADR Index ...
7. Author: Once it's ready for review (ideally this is clearly defined in some guidelines or docs), update the status to "in review" and mark the pull request as "ready for review"
8. Reviewer: reviews ... (ideally we link to some guidelines or docs)
9. Author: addresses any feedback..
10. Reviewer: Updates the status to "accepted" and merges.
11. (optional) Author: If "withdrawn" update the status respectively and close the PR.
12. (optional) Reviewer: If it becomes clear (e.g. over the course of multiple rounds of review) that an ADR is "unsuitable", update the status respectively and close the PR

I could imagine that it may more sense in the long run for there to be one set of instructions for the author and another for the reviewer; however, in the spirit of iteration and to try to keep things simple for now, I think it's fine to keep them combined.

Reply:
bump?

Reply:
+1 to everything here:
1. Feels like a "good enough" iteration as we figure it out
2. I like the clarity
3. NIT: Bold the responsible stakeholder